style.css

```diff
@@ -1069,4 +1096,37 @@ textarea {
     margin-top: 2%;
     padding-bottom: 2%;
   }
+
+  nav .mainMenu {
+    width: 90%;
+    height: 100%;
+    position: fixed;
+    right: 0 px;
```

Comment:
- [ ] The close button on the mobile navigation is too close to the edge. This is because you have set the padding to 0.
Please fix this so that your impelemntation matches the Figma.

**Figma**
![screen2](https://github.com/MayPyone/My_Portfolio/assets/99656687/ee4aff92-fd54-4ea2-b4ff-176bd0f13a0b)

**Your design**
![close2](https://github.com/MayPyone/My_Portfolio/assets/99656687/15a4293e-bb5c-485d-9707-840b2e807f1d)